lib/actions/preload.js

```diff
@@ -39,6 +39,9 @@ const applicationExpandOptions = {
 			status: 'success',
 		},
 	},
+	should_be_running__release: {
+		$expand: 'commit',
```

Comment:
This would be the cause @pdcastro

Reply:
I assume it wasn't caught by typescript because js files aren't as strongly typed